src/config.rs

```diff
@@ -101,7 +101,9 @@ fn default_requires_worktree() -> bool {
 }
 
 // This implementation is only valid for Tests among those registered for a single Manager.
-impl GraphNode<String> for Test {
+impl GraphNode for Test {
+    type I = String;
```

Comment:
`I` would be a name for a type parameter, but assocated types get proper names: https://doc.rust-lang.org/nightly/style-guide/items.html#associated-types

So let's use `NodeId`

(If you're using VSCode, you should be able to just use the rename command to do this through the whole codebase at once, for me the shortcut is F2)

Reply:
Ha, this is exactly the kind of thing that I hoped to learn! Done.